tests/unittests/Foundation/NSLockTests.m

```diff
@@ -19,66 +19,6 @@
 #import <thread>
 #import <mutex>
 #import <condition_variable>
-#import <chrono>
-#import "NSLock+Internal.h"
-
-TEST(NSLock, initWithName) {
```

Comment:
_lockCount is our private function

---
In reply to: [72314890](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72314890) [](ancestors = 72314890)